utilities/roslz4/src/lz4s.c

```diff
@@ -573,7 +573,10 @@ int roslz4_buffToBuffCompress(char *input, unsigned int input_size,
 
   int ret;
   ret = roslz4_compressStart(&stream, block_size_id);
-  if (ret != ROSLZ4_OK) { return ret; }
+  if (ret != ROSLZ4_OK) {
+    roslz4_decompressEnd(&stream);
```

Comment:
This doesn't look symmetric should it be `roslz4_compressEnd`?

Reply:
Ah Yes! Thanks for catching this